### ISSUE TITLE
"Appears in X Checklist datasets:" has disappeared

### ISSUE THREAD

**Issue by MattBlissett** [gbif/portal16#1376]
"Appears in X Checklist datasets:" has disappeared.

See https://www.gbif.org/species/2435350, for example.

**Comment by MortenHofft** [comment]
@thomasstjerne you know the species pages better than I, so feel free to elaborate.

I see that it does a call to 
http://api.gbif.org/v1/dataset/search?taxonKey=2435350&type=CHECKLIST
which returns zero results.

I assume the reason it no longer shows on species pages is because that feature is no longer supported by the dataset search API?

I do not see the taxonKey param in the API documentation.

**Comment by MattBlissett** [comment]
I haven't looked at what the registry used to provide, but is not https://api.gbif.org/v1/species/2435350/related the appropriate query? (@mdoering?)

**Comment by mdoering** [comment]
@MattBlissett yes, that does return usages with the same backbone key across all datasets.

But I do believe we had taxon keys in the registrys solr dataset index before as you can see in [DatasetSearchParameter.java](https://github.com/gbif/gbif-api/blob/gbif-api-0.109/src/main/java/org/gbif/api/model/registry/search/DatasetSearchParameter.java#L92): 

```
  /**
   * Backbone name usage key that this dataset covers.
   */
  TAXON_KEY(Integer.class),
```

There is (was?) a CLB [dataset analysis cli](https://github.com/gbif/checklistbank/blob/checklistbank-2.82/checklistbank-cli/src/main/java/org/gbif/checklistbank/cli/analysis/AnalysisService.java#L42) running that [updated the dataset solr index](https://github.com/gbif/checklistbank/blob/checklistbank-2.82/checklistbank-cli/src/main/java/org/gbif/checklistbank/cli/analysis/DatasetIndexUpdater.java#L43). Has this index been dropped?

**Comment by mdoering** [comment]
At least the dataset index is still on C5 and the analysis cli running.

**Comment by mdoering** [comment]
@asturcon @MattBlissett how does the registry answer such taxon_key searches now? Should the CLB code be updated to index into ES instead?
